integration_tests/cli/auth-demo.sh

```diff
@@ -484,7 +496,164 @@ echo "Get $user account"
+result=$(dcld query auth revoked-account --address=$user_address)
+check_response "$result" "\"address\": \"$user_address\""
+
+test_divider
+
+echo "Bob rejects account for \"$user\""
+result=$(echo $passphrase | dcld tx auth reject-add-account --address="$user_address" --info="Bob is rejecting this account" --from bob --yes)
+check_response "$result" "\"code\": 0"
+
+test_divider
+
+echo "Bob cannot reject the same account \"$user\" for the second time"
+result=$(echo $passphrase | dcld tx auth reject-add-account --address="$user_address" --info="Bob is rejecting this account" --from bob --yes 2>&1 || true)
+response_does_not_contain "$result" "\"code\": 0"
+
+test_divider
+
+echo "Get all rejected accounts. $user account must be exist in the list, because enough rejected approvals received"
+result=$(dcld query auth all-rejected-accounts)
+check_response "$result" "\"address\": \"$user_address\""
+
+test_divider
+
+echo "Get all proposed accounts. $user account doesn't exist in the list"
+result=$(dcld query auth all-proposed-accounts)
+response_does_not_contain "$result" "\"address\": \"$user_address\""
+
+test_divider
+
+echo "Get all accounts. $user account doesn't exist in the list"
+result=$(dcld query auth all-accounts)
+response_does_not_contain "$result" "\"address\": \"$user_address\""
+
+test_divider
+
+echo "Get all proposed accounts to revoke. No $user account in the list"
+result=$(dcld query auth all-proposed-accounts-to-revoke)
+check_response "$result" "\[\]"
+
+test_divider
+
+echo "Get all revoked accounts to revoke. No $user account in the list"
+result=$(dcld query auth all-proposed-accounts-to-revoke)
+response_does_not_contain "$result" "\"address\": \"$user_address\""
+
+test_divider
+
+echo "Get a rejected account for $user. $user account must exist"
+result=$(dcld query auth rejected-account --address=$user_address)
+check_response "$result" "\"address\": \"$user_address\""
+check_response_and_report "$result"  $jack_address "json"
+check_response_and_report "$result"  '"info": "Jack is proposing this account"' "json"
+check_response_and_report "$result"  $alice_address "json"
+check_response_and_report "$result"  '"info": "Alice is rejecting this account"' "json"
+check_response_and_report "$result"  $bob_address "json"
+check_response_and_report "$result"  '"info": "Bob is rejecting this account"' "json"
+
+test_divider
+
+echo "Get a account for $user is not found"
+result=$(dcld query auth account --address=$user_address)
+check_response "$result" "Not Found"
+
+test_divider
+
+echo "Get a proposed account to revoked for $user is not found"
+result=$(dcld query auth proposed-account-to-revoke --address=$user_address)
+check_response "$result" "Not Found"
+
+test_divider
+
+echo "Get a revoked account for $user is not found"
+result=$(dcld query auth revoked-account --address=$user_address)
+check_response "$result" "\"address\": \"$user_address\""
 
+test_divider
```

Comment:
I believe we should also check whether a rejected account is removed from rejected list once it is proposed again.
The same for all other proposals.

Reply:
Done! ✅